src/Checkbox/index.js

```diff
@@ -0,0 +1,38 @@
+const React = require('react');
+const { getRestProps } = require('../utils');
+const _ = require('lodash');
+
+const styles = require('./Checkbox.scss');
+
+const propTypes = {
+  id: React.PropTypes.string,
```

Comment:
might be a bit harder to debug if the unique id is random and not semantic
your call. not a huge deal.